libs/ui/src/visual_mode_disabled_overlay.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import { DisplaySettingsManagerContext } from './display_settings_manager_context';
+import { Button } from './button';
+import { appStrings } from './ui_strings';
+import { H2 } from './typography';
+import { useScreenInfo } from './hooks/use_screen_info';
+import { Header } from './display_settings/header';
+
+const Overlay = styled.div`
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  z-index: 1000;
+  left: 0;
+  top: 0;
+  background: #fff;
+  padding: 1em;
+`;
+
+const ButtonContainer = styled.div`
+  display: flex;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+`;
+
+export function VisualModeDisabledOverlay(): JSX.Element {
+  const displaySettingsManager = React.useContext(
+    DisplaySettingsManagerContext
+  );
+  const screenInfo = useScreenInfo();
+  return (
+    <Overlay aria-hidden>
+      <Header portrait={screenInfo.isPortrait}>
```

Comment:
Not sure if this is necessary. Using the factored-out `display_settings/header` component keeps the header consistent with the display settings page, but it might be overkill since we are guaranteed mark-scan screen orientation can't change.